.drone.yml

```diff
@@ -1,38 +1,60 @@
-build:
-  image: golang:1.5
+workspace:
+  base: /go
+  path: src/github.com/drone-plugins/drone-pypi
+
+pipeline:
+  test:
+    image: golang:1.10
+    pull: true
+    commands:
+      - go vet ./...
+      - go test -cover ./...
+      - rm -rf dist
+      - rm -rf drone_pypi_test.egg-info
   environment:
-    - GO15VENDOREXPERIMENT=1
-    - GOOS=linux
-    - GOARCH=amd64
-    - CGO_ENABLED=0
-    - DRONE_PYPI_PATH=testdata
-    - DRONE_PYPI_REPOSITORY=http://localhost:8000
-    - DRONE_PYPI_DISTRIBUTIONS=sdist
-  commands:
-    - apt-get update && apt-get install -y python-setuptools
-    - go get
-    - go build
-    - go test -cover
+    - PLUGIN_REPOSITORY=https://test.pypi.org/legacy/
```

Comment:
Can't we just start a simple pypi server locally as part of the service definition? Presse write the environment block before the commands block :)

Reply:
Good idea, definitely possible. Will do this after Christmas. Merry Christmas to all of you also.

Reply:
Done